### PR TITLE
Add/split subset

### PR DESCRIPTION
**Pull request by shibuiwilliam**
# What
- Add README for pytest.
- Add split subset command to some README.

### CODE REVIEW

File: gradle/README.md [hunk 1]
```diff
@@ -92,6 +92,31 @@ See https://docs.gradle.org/7.0.2/userguide/command_line_interface.html#sec:comm
 
 BUILD SUCCESSFUL in 1s
 3 actionable tasks: 3 executed
+
+
+# Apply `--split` option to split the subset into multiple bins to run simultaneously in parallel test environment.
+$ launchable subset --target ${CONFIDENCE} --build ${BUILD_NAME} --split gradle src/test/java
```
[the remark this answers is not in the document]
Comment (shibuiwilliam) [reply gradle/README.md:98]: fixed.
https://github.com/launchableinc/examples/pull/53/commits/ddb8b53520bd0d60ccc868c644c906106ffe86b3

File: pytest/README.md [hunk 2]
```diff
@@ -1,23 +1,205 @@
...
 
 ```shell
-$  pytest --collect-only  -q
-tests/funcs3_test.py::test_func4
-tests/funcs3_test.py::test_func5
+$ BUILD_NAME=test
+$ CONFIDENCE="50%"
+
+
+$ launchable subset --target ${CONFIDENCE} --build ${BUILD_NAME} pytest . > subset.txt
```
[the remark this answers is not in the document]
Comment (shibuiwilliam) [reply pytest/README.md]: fixed for ${TARGET}
https://github.com/launchableinc/examples/pull/53/commits/ddb8b53520bd0d60ccc868c644c906106ffe86b3

File: pytest/README.md [hunk 3]
```diff
@@ -1,23 +1,205 @@
...
+--------------------------- generated xml file: /Users/shibuiyuusuke/launchable/samples/examples/pytest/report.xml ---------------------------
+========================================================== short test summary info ===========================================================
+FAILED tests/funcs3_test.py::test_func5 - assert 1 == False
+FAILED tests/test_func5.py::test_func5[6*9-42] - AssertionError: assert 54 == 42
+FAILED tests/test_funcs1.py::test_func2 - assert 1 == False
+======================================================== 3 failed, 12 passed in 4.18s ========================================================
+[22-10-06 12:20:45] shibuiyuusuke:~/launchable/samples/examples/pytest
+
+
+$ launchable record tests --build ${BUILD_NAME} pytest .
```
[the remark this answers is not in the document]
Comment (shibuiwilliam) [reply pytest/README.md]: fix. made a report directory with a report.xml.
https://github.com/launchableinc/examples/pull/53/commits/45d680dbe5fc8cadffa6e29de7560268c3a53354

File: pytest/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (shibuiwilliam) [reply pytest/README.md]: @Konboi 
> Was this command no longer needed?

Is it?

**Review by Konboi — APPROVED** [review]

**Pull request merged**